text/0043-phrase-query.md

```diff
@@ -0,0 +1,172 @@
+
+- It could have had a specific search field only dedicated to this ISBN field in order to make it a filter and using the `=` operator. Not great for the UX.
+
+- It could also have kept a single search field and detected that an ISBN was entered in the search field using a regex to inject that filter at that point. Not great for the developper experience, moreover, what happens when a pattern cannot be specifically determined?
+
+The Phrase Query feature will easily solve this case but will also adapt to the needs of the user performing the search.
+
+### III. Additional Materials
+
+#### Algolia
+
+Algolia allows the use of Phrase Query syntax as long as the `advancedSyntax` parameter is set to true in the settings.
+
+As Algolia documentation said, a phrase query represents a specific sequence of terms that must be matched next to one another. A phrase query needs to be surrounded by double quotes ("). For example, the query "search engine" only returns a record if it contains “search engine” exactly in at least one attribute.
+
+#### Elasticsearch
+
+Elasticsearch provides a `match_phrase` field to perform this type of search.
+
+```
+GET /_search
+{
+  "query": {
+    "match_phrase": {
+      "message": "this is a test"
+    }
+  }
+}
+```
+
+### IV. Explanation
+
+Let's say I want to search for a specific book with a title strictly containing `Plays and Playwrights 2002`.
+
+Using the standard query parameter syntax as `q` equals to `Plays and Playwrights 2002`, will lead to have multiple search results because of the ranking rules criterions.
+
+```
+{
+    "hits": [
+        {
+            "title": "Plays and Playwrights 2002",
+            "author": "Martin Denton"
+        },
+        {
+            "title": "Plays and Playwrights 2009",
+            "author": "Martin Denton"
+        },
+        {
+            "title": "Plays and Playwrights 2008",
+            "author": "Robert Attweiler"
+        },
+        ...
+        {
+            "title": "The Best American Short Plays 2006-2007",
+            "author": "Barbara Parisi"
+        },
+        {
+            "title": "New Playwrights: The Best Plays of 2000",
+            "author": "D. L. Lepidus"
+        },
+        {
+            "title": "Women Heroes: Six Short Plays from the Women's Project",
+            "author": "Julia Miles"
+        }
+    ],
+    "nbHits": 31,
+    "exhaustiveNbHits": false,
+    "query": "Plays and Playwrights 2002",
+    "limit": 20,
+    "offset": 0,
+    "processingTimeMs": 7
+}
+```
+
+To use the Phrase Query syntax, simply surround the contiguous search terms with the characters `"`.
+
+Using the Phrase Query syntax this way, with `q` equals to `"Plays and Playwrights 2002"`, will lead to have only one result because the title is exactly written like that.
```

Comment:
```suggestion
Using the Phrase Query syntax this way, with `q` equals to `"Plays and Playwrights 2002"`, will lead to have only one result because the title is written exactly like that.
```